pilot/prompts/prompts.py

```diff
@@ -89,7 +89,7 @@ def generate_messages_from_description(description, app_type, name):
       ]
     """
     # "I want you to create the app {name} that can be described: ```{description}```
-    prompt = get_prompt('high_level_questions/specs.prompt', {
+    prompt = get_prompt('high_level_questions/specs.prompt', original_data={
```

Comment:
I think this might have been addressed `def get_prompt(prompt_name, model=None, original_data=None): ` original_data now has a default value. I am not 100% can you please check